cmds/pxeboot/pxeboot.go

```diff
@@ -114,29 +131,13 @@ func Boot(lease dhclient.Lease) error {
 	if p4, ok := lease.(*dhclient.Packet4); ok {
 		ip = p4.Lease().IP
 	}
-	img, err := getBootImage(uri, lease.Link().Attrs().HardwareAddr, ip)
-	if err != nil {
-		return err
-	}
-	log.Printf("Got configuration: %s", img)
-
-	if *dryRun {
-		img.ExecutionInfo(log.New(os.Stderr, "", log.LstdFlags))
-		return nil
-	} else if err := img.Execute(); err != nil {
-		return fmt.Errorf("kexec of %v failed: %v", img, err)
-	}
-
-	// Kexec should either return an error or not return.
-	panic("unreachable")
+	return getBootImage(uri, lease.Link().Attrs().HardwareAddr, ip)
 }
 
 func main() {
 	flag.Parse()
 
-	ctx, cancel := context.WithTimeout(context.Background(), dhcpTries*dhcpTimeout)
-	defer cancel()
-	if err := Netboot(ctx, "eth0"); err != nil {
+	if err := Netboot("eth0"); err != nil {
```

Comment:
will the interface become a flag at a later stage?

Reply:
Yeah, soon enough, another PR...

Reply:
I plan to make it the same thing as in dhclient.